openpype/hosts/traypublisher/plugins/create/create_editorial.py

```diff
@@ -682,6 +683,7 @@ def _get_base_instance_data(
         # create creator attributes
         creator_attributes = {
             "asset_name": shot_name,
+            "Folder path": shot_metadata["folderPath"],
             "Parent hierarchy path": shot_metadata["hierarchy"],
```

Comment:
I do expect `Folder path` and `Parent hierarchy path` are there so they're showed in UI, not as value. In that case in OpenPype there should be only `Parent hierarchy path` and `Folder path` only in AYON.

Reply:
implemented here 62af7138e11cc11509b91ec2f1ee7cc1995fb10b